packages/ove-service-layout/src/index.js

```diff
@@ -25,9 +27,9 @@ app.post("/render", renderRoute);
 // eslint-disable-next-line no-unused-vars
 app.use((error, req, res, next) => {
```

Comment:
Can't we mark unused vars with an underscore infront of it and get rid of this complain from the linter? I thought we managed to do that successfully in OVE core.

Reply:
I know this is a pythonic convention for ignored vars, but in js worlds there also a library called underscore, which is abbreviated by _. I find that using the convention of _ for ignored vars to be incredibly confusing in that situation so that is why I prefer to disable linting on a case by case basis.

Reply:
@senakafdo @jamesscottbrown @davidbirchwork @DavidAkroyd That's not quite right and if you did this change in OVE you should definitely change it back.

According to the express documentation, the error handler **always has 4 parameters**. Otherwise is interpreted as something else, e.g. a regular request handler: https://expressjs.com/en/4x/api.html#app.use

So in this case, if it's just a matter of style, I'd rather keep this code as it is.

Reply:
Ah, I did not realise that. We handle all errors generated within OVE and also the apps, and for anything out of control we let express handle it in the standard way, so we don't have an explicit error handler implementation. The fixes that were done as per request of @jamesscottbrown and @davidbirchwork were not the ones that involved an error handling middleware. Also, express AFAIU lets you define a global error handler, so even if we ever had such it would be that.
Given this is the reason behind this, you cannot simplify this further. Anyway I'm sorry that I wasn't aware of this funny behaviour of express earlier. It is very easy for them to test the type of the argument(s) passed rather than implementing this in such a sloppy way.

Reply:
@oserban, FTR, please also find [an article](https://thecodebarbarian.com/80-20-guide-to-express-error-handling) that explains why express error handling is so poorly designed, :), along with some workaround to deal with async/await etc.

Reply:
Yes, agreed. BTW, I have already approved this PR 4 days ago, this was a conversation related to the best practice and not around any issues in the PR itself. That too was resolved following your explanation. 
Of course we have a convention (python-like - which is what the linter was happy with) in OVE and OVE-Apps (at least for the apps I wrote) but that doesn't mean it needs to be followed by all.